inference-engine/src/legacy_api/include/legacy/graph_transformer.h

```diff
@@ -9,7 +9,7 @@
 
 #pragma once
 
-#include <caseless.hpp>
+#include <caseless.hpp>  // Plugin API
```

Comment:
Is it really necessary? I mean, if it worths to mention, maybe include itself should be in a form of `#include <plugin_api/caseless.hpp>`?

Reply:
Removed.